src/deploy/hash_fns.js

```diff
@@ -7,25 +7,17 @@ const pump = promisify(require('pump'))
 
 const { hasherCtor, manifestCollectorCtor } = require('./hasher_segments')
 
-const hashFns = async (dir, opts) => {
-  opts = {
-    concurrentHash: 100,
-    assetType: 'function',
-    hashAlgorithm: 'sha256',
-    // tmpDir,
-    statusCb: () => {},
-    ...opts,
-  }
+const hashFns = async (dir, { tmpDir, concurrentHash, hashAlgorithm = 'sha256', assetType = 'function', statusCb }) => {
```

Comment:
`concurrentHash` and `statusCb` default values are assigned by the calling function.